javascript/chronometer.js

```diff
@@ -1,26 +1,64 @@
 class Chronometer {
   constructor() {
+    this.currentTime = 0
+    this.intervalId = 0
+  
     // ... your code goes here
   }
   startClick(callback) {
+    
+    let intId = setInterval(()=>{
```

Comment:
you created an attribute for the intervalId in line 4, so it is better to save the interval id inside that attribute instead of a simple variable, the reason is that this variable only exists inside this scope, so you won't be able to access if later to clear it.
`this.intervalId = setInterval(()=>{...`